docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -749,6 +827,2115 @@
     "plt.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "002b5c74",
+   "metadata": {},
+   "source": [
+    "### Visualizing the answer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e2b6b22",
+   "metadata": {},
+   "source": [
+    "In order to reduce computational costs we have reduced the problem's qubit operator to the minimum ammount of qubits needed to represent the shape of the protein. In order to decode the answer we need to understand how this has been done.\n",
+    "* The shape of the protein has been encoded by a sequence of turns , $\\{0,1,2,3\\}$. Each turn represents a different direction the lattice.\n",
+    "* For a main bead of $N_{aminoacids}$ aminoacids in a lattice, we need $N_{aminoacids}-1$ turns in order to represent its shape. However, the orientation of the protein is not relevant to its energy. Therefore the first two turns of the shape can be set to $[1,0]$ without loss of generality.\n",
+    "* If the second bead doesn't have any side chain, we can also set the $6^{th}$ qubit to $[1]$ without breaking symmetry.\n",
+    "* Since the lenght of the secondary chains is always $1$ we only need one turn to describe the shape of the chain.\n",
+    "\n",
+    "The total ammount of qubits we need to represent the shape of the protein will be $2(N_{aminoacids}-3)$ if there is a secondary chain comming out of the second bead or $2(N_{aminoacids}-3) - 1$ else. All the other qubits will be unused during the optimization process. See:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "49a65992",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The bitstring representing the shape of the protein during optimization is:  101100011\n",
+      "The expanded expression is: 1******0*****************************************************************************************************************************110001*1****\n"
```

Comment:
Hi, I made that choice when I was implementing the thing. I did not put much thinking into this and I am fine with alternatives we consider better. Another option, more in the regex spirit, would be `.`.  The values of `*` qubits do not contribute to the final result so their actual value might be anything, hence the dot that means any character.

Reply:
I think `.` could be nice, too. I'm open to either `_` or `.`.
I think changing this would be good though, because both of those options "appear nicer" because they are comparatively small:
```
1***0***1**
1___0___1__
1...0...1..
```
I find the current `*` option to be quite busy when viewing them side-by-side like this

Reply:
`_` might not be ideal because it is difficult to count many occurrences as they connect to each other: `___________`.